fastlane/lanes/localization.rb

```diff
@@ -5,11 +5,11 @@
 #################################################
 
 # URL of the GlotPress project containing the app's strings
-GLOTPRESS_APP_STRINGS_URL = 'https://translate.wordpress.org/projects/apps/ios/dev/'
+GLOTPRESS_APP_STRINGS_PROJECT_URL = 'https://translate.wordpress.org/projects/apps/ios/dev/'
 
 # URL of the GlotPress projects containing App Store metadata (title, keywords, release notes, …)
-GLOTPRESS_WORDPRESS_METADATA_PROJECT_URL = 'https://translate.wordpress.org/projects/apps/ios/release-notes/'
-GLOTPRESS_JETPACK_METADATA_PROJECT_URL = 'https://translate.wordpress.com/projects/jetpack/apps/ios/release-notes/'
+GLOTPRESS_WORDPRESS_APP_STORE_METADATA_PROJECT_URL = 'https://translate.wordpress.org/projects/apps/ios/release-notes/'
+GLOTPRESS_JETPACK_APP_STORE_METADATA_PROJECT_URL = 'https://translate.wordpress.com/projects/jetpack/apps/ios/release-notes/'
```

Comment:
The thought occurred to me to use 

```ruby
GLOTPRESS_APP_STORE_METADATA_PROJECT_URL_WORDPRESS = ...
GLOTPRESS_APP_STORE_METADATA_PROJECT_URL_JETPACK_ = ...
```

That would have made the `_` distribution nicer to look at. I think "GlotPress `<app name>` App Store metadata project URL reads better, though.

Reply:
Yeah that's also why I included `…_FOR_X` variants in my list of examples/suggestions in the other PR, for nice alignment/distribution of the `_` in the constants as well … but ultimately I wasn't super happy with that `_FOR_…` vibe either … so I agree this one reads better after all.